src/VoyagerServiceProvider.php

```diff
@@ -117,7 +117,11 @@ protected function registerViewComposers()
      */
     protected function addStorageSymlinkAlert()
     {
-        $currentRouteAction = app('router')->current()->getAction();
+        if (app('router')->current() !== null) {
+            $currentRouteAction = app('router')->current()->getAction();
```

Comment:
You should cache the instantiation of `app('router')` in my opinion. Even `$currentRoute = app('router')->current()` if that's the only use you're giving to it in that function.